platform/app/src/routes/buildModeRoutes.tsx

```diff
@@ -45,21 +46,23 @@ export default function buildModeRoutes({
       const path = `/${mode.routeName}/${dataSourceName}`;
 
       // TODO move up.
-      const children = () => (
+      const children = (...props) => (
```

Comment:
Correct me if I am wrong, but the `ModeRoute` at present really does not care about `props`, so consider not including them here. Or am I missing something?

Reply:
Updated! thanks :)